package.json

```
@@ -52,7 +52,7 @@
       "type": "object",
       "properties": {}
     },
-    "autoToggle": {
+    "showMinimapOnAtomStart": {
```

Actually it might be better just to change the text in the settings view instead of the setting name.
```suggestion
    "autoToggle": {
      "title": "Show Minimap On Atom Start",
```

---

Then you can close #729 since the code wouldn't have to change.